Lib/test/test_pydoc.py

```diff
@@ -123,7 +123,7 @@ class C(builtins.object)
     c_alias = test.pydoc_mod.C[int]
     list_alias1 = typing.List[int]
     list_alias2 = list[int]
-    type_union1 = typing.Union[int, str]
+    type_union1 = int | str
     type_union2 = int | str
```

Comment:
Does this imply that there are now two tests that do the same thing?

Reply:
Kind of, `type_union1` is `Union[int, str]` and `type_union2` is `int | str`. Those now evaluate to the same object, but pydoc doesn't need to know about that, so I think it still makes sense to test both.

Reply:
I don't quite understand this. After your changes here, every occurrence of `type_union1` and `type_union2` in this file are now identical. How is `type_union1` still testing `Union[int, str]` after your changes?

Reply:
It's defined in `Lib/test/pydoc_mod.py`.